Test failure on nightly

```julia
`inspect_session`: Error During Test at /home/runner/work/ExplicitImports.jl/ExplicitImports.jl/test/runtests.jl:1028
  Got exception outside of a @test
  BoundsError: attempt to access 0-element Vector{Module} at index [1]
  Stacktrace:
    [1] throw_boundserror(A::Vector{Module}, I::Tuple{Int64})
      @ Base ./essentials.jl:14
    [2] getindex
      @ ./essentials.jl:909 [inlined]
    [3] first
      @ ./abstractarray.jl:452 [inlined]
    [4] choose_exporter
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:141 [inlined]
    [5] #212
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:241 [inlined]
    [6] lt(o::Base.Order.By{ExplicitImports.var"#212#213", Base.Order.Lt{ExplicitImports.var"#lt#211"}}, a::@NamedTuple{name::Symbol, source::Module, exporters::Vector{Module}, location::String}, b::@NamedTuple{name::Symbol, source::Module, exporters::Vector{Module}, location::String})
      @ Base.Order ./ordering.jl:120
    [7] _sort!(v::Vector{@NamedTuple{name::Symbol, source::Module, exporters::Vector{Module}, location::String}}, ::Base.Sort.InsertionSortAlg, o::Base.Order.By{ExplicitImports.var"#212#213", Base.Order.Lt{ExplicitImports.var"#lt#211"}}, kw::@NamedTuple{scratch::Nothing, lo::Int64, hi::Int64})
      @ Base.Sort ./sort.jl:842
    [8] _sort!
      @ ./sort.jl:797 [inlined]
    [9] _sort!
      @ ./sort.jl:744 [inlined]
   [10] _sort!
      @ ./sort.jl:683 [inlined]
   [11] _sort!
      @ ./sort.jl:561 [inlined]
   [12] #sort!#23
      @ ./sort.jl:1711 [inlined]
   [13] sort!
      @ ./sort.jl:1704 [inlined]
   [14] explicit_imports_nonrecursive(mod::Module, file::String; skip::Tuple{Module, Module, Module}, strict::Bool, warn_stale::Nothing, file_analysis::ExplicitImports.FileAnalysis)
      @ ExplicitImports ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:[242](https://github.com/ericphanson/ExplicitImports.jl/actions/runs/11163138707/job/31029364712#step:6:245)
   [15] explicit_imports_nonrecursive
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:191 [inlined]
   [16] #187
      @ ./none:-1 [inlined]
   [17] iterate
      @ ./generator.jl:48 [inlined]
   [18] collect_to!(dest::Vector{Pair{Module, Vector{@NamedTuple{name::Symbol, source::Module, exporters::Vector{Module}, location::String}}}}, itr::Base.Generator{Vector{Pair{Module, String}}, ExplicitImports.var"#187#188"{Tuple{Module, Module, Module}, Bool, Nothing, Dict{String, ExplicitImports.FileAnalysis}}}, offs::Int64, st::Int64)
      @ Base ./array.jl:847
   [19] collect_to_with_first!
      @ ./array.jl:825 [inlined]
   [20] collect(itr::Base.Generator{Vector{Pair{Module, String}}, ExplicitImports.var"#187#188"{Tuple{Module, Module, Module}, Bool, Nothing, Dict{String, ExplicitImports.FileAnalysis}}})
      @ Base ./array.jl:799
   [21] explicit_imports(mod::Module, file::String; skip::Tuple{Module, Module, Module}, strict::Bool, warn_stale::Nothing, file_analysis::Dict{String, ExplicitImports.FileAnalysis})
      @ ExplicitImports ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:131
   [22] explicit_imports
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:118 [inlined]
   [23] print_explicit_imports(final_io::IOBuffer, mod::Module, file::String; skip::Tuple{Module, Module, Module}, warn_implicit_imports::Bool, warn_improper_explicit_imports::Nothing, warn_improper_qualified_accesses::Bool, report_non_public::Bool, strict::Bool, show_locations::Bool, separate_lines::Bool, linewidth::Int64, allow_internal_accesses::Bool, allow_internal_imports::Bool, warn_stale::Nothing, recursive::Bool, name_fn::ExplicitImports.var"#print_explicit_imports##0#print_explicit_imports##1")
      @ ExplicitImports ~/work/ExplicitImports.jl/ExplicitImports.jl/src/interactive_usage.jl:70
   [24] print_explicit_imports (repeats 2 times)
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/src/interactive_usage.jl:43 [inlined]
   [25] inspect_session(io::IOBuffer; skip::Tuple{Module, Module}, inner::typeof(print_explicit_imports))
      @ ExplicitImports ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:394
   [26] inspect_session(io::IOBuffer)
      @ ExplicitImports ~/work/ExplicitImports.jl/ExplicitImports.jl/src/ExplicitImports.jl:389
   [27] sprint(::Function; context::Nothing, sizehint::Int64)
      @ Base ./strings/io.jl:117
   [28] sprint
      @ ./strings/io.jl:110 [inlined]
   [29] #189
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/test/runtests.jl:1031 [inlined]
   [30] with_logstate(f::var"#189#190", logstate::Base.CoreLogging.LogState)
      @ Base.CoreLogging ./logging/logging.jl:522
   [31] with_logger(f::Function, logger::Base.CoreLogging.NullLogger)
      @ Base.CoreLogging ./logging/logging.jl:632
   [32] top-level scope
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/test/runtests.jl:1030
   [33] macro expansion
      @ ~/.julia/juliaup/julia-nightly/share/julia/stdlib/v1.12/Test/src/Test.jl:1703 [inlined]
   [34] macro expansion
      @ ~/work/ExplicitImports.jl/ExplicitImports.jl/test/runtests.jl:1030 [inlined]
   [35] include(fname::String)
      @ Main ./sysimg.jl:38
   [36] top-level scope
      @ none:6
   [37] eval
      @ ./boot.jl:439 [inlined]
   [38] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:295
   [39] _start()
      @ Base ./client.jl:558
Test Summary:     | Error  Total   Time
`inspect_session` |     1      1  16.4s
ERROR: LoadError: Some tests did not pass: 0 passed, 0 failed, 1 errored, 0 broken.
in expression starting at /home/runner/work/ExplicitImports.jl/ExplicitImports.jl/test/runtests.jl:1028
Package ExplicitImports errored during testing
Error: Process completed with exit code 1.
```

seen in #87, #86 etc

--- a/Project.toml
+++ b/Project.toml
@@ -1,7 +1,7 @@
 name = "ExplicitImports"
 uuid = "7d51a73a-1435-4ff3-83d9-f097790105c7"
 authors = ["Eric P. Hanson"]
-version = "1.10.1"
+version = "1.10.2"
 
 [deps]
 AbstractTrees = "1520ce14-60c1-5f80-bbc7-55ef81b5835c"

--- a/src/find_implicit_imports.jl
+++ b/src/find_implicit_imports.jl
@@ -67,7 +67,15 @@ function find_implicit_imports(mod::Module; skip=(mod, Base, Core))
                     push!(es, e)
                 end
             end
-            mod_lookup[name] = (; source, exporters=es)
+            # if there are no matches (empty `es`), we will skip it
+            # This seemed to happen for `tryparse` in `Pkg.Types` which resolves to `Base.tryparse`
+            # and does not match `TOML.tryparse` which was the only candidate to compare to
+            # (since we want to skip `Base.tryparse` as `Base` is in `skip`)
+            # If there are no matches, such as in this case, we don't want to count it
+            # as an implicit import, since it is probably only from a module in `skip`.
+            if !isempty(es)
+                mod_lookup[name] = (; source, exporters=es)
+            end
         end
     end
     return mod_lookup

--- a/test/examples.jl
+++ b/test/examples.jl
@@ -57,3 +57,19 @@ using UUIDs
 
 v = UUID[]
 end
+
+module ModWithTryparse
+using Main: @public_or_export
+
+@public_or_export tryparse
+function tryparse()
+    return 1
+end
+end
+# https://github.com/ericphanson/ExplicitImports.jl/issues/88
+module Mod88
+
+using ..ModWithTryparse
+
+tryparse
+end

--- a/test/examples.jl
+++ b/test/examples.jl
@@ -59,9 +59,9 @@ v = UUID[]
 end
 
 module ModWithTryparse
-using Main: @public_or_export
+using Main: @public
 
-@public_or_export tryparse
+@public tryparse
 function tryparse()
     return 1
 end

--- a/test/public_compat.jl
+++ b/test/public_compat.jl
@@ -4,8 +4,16 @@
     macro public_or_export(symbol::Symbol)
         return esc(Expr(:public, symbol))
     end
+    macro public(symbol::Symbol)
+        return esc(Expr(:public, symbol))
+    end
+
 else
     macro public_or_export(symbol::Symbol)
         return esc(Expr(:export, symbol))
     end
+    macro public(symbol::Symbol)
+        return nothing
+    end
+
 end

--- a/test/runtests.jl
+++ b/test/runtests.jl
@@ -601,6 +601,10 @@ end
     @test contains(str, "• qr")
 end
 
+@testset "Handle public symbols with same name as exported Base symbols (#88)" begin
+    statements = using_statement.(explicit_imports_nonrecursive(Mod88, "examples.jl"))
+
+end
 @testset "Don't skip source modules (#29)" begin
     # In this case `UUID` is defined in Base but exported in UUIDs
     ret = ExplicitImports.find_implicit_imports(Mod29)[:UUID]

--- a/test/runtests.jl
+++ b/test/runtests.jl
@@ -603,6 +603,7 @@ end
 
 @testset "Handle public symbols with same name as exported Base symbols (#88)" begin
     statements = using_statement.(explicit_imports_nonrecursive(Mod88, "examples.jl"))
+    @test statements == "using .ModWithTryparse: ModWithTryparse"
 
 end
 @testset "Don't skip source modules (#29)" begin

--- a/test/runtests.jl
+++ b/test/runtests.jl
@@ -603,7 +603,7 @@ end
 
 @testset "Handle public symbols with same name as exported Base symbols (#88)" begin
     statements = using_statement.(explicit_imports_nonrecursive(Mod88, "examples.jl"))
-    @test statements == "using .ModWithTryparse: ModWithTryparse"
+    @test statements ==  ["using .ModWithTryparse: ModWithTryparse"]
 
 end
 @testset "Don't skip source modules (#29)" begin

